LICENSE

```diff
@@ -0,0 +1,22 @@
+This Cliqz Browser is building upon the Mozilla open source project,
```

Comment:
@chrmod maybe we call it "This Cliqz browser for Android" ?

Reply:
@luciancor Well, I'll suggest to just say `The Cliqz Browser for Android is....` otherwise `This Cliqz Browser` because it sounds like `This specific Cliqz Browser, among the many we develop, is building...`.